src/dmd/cli.d

```diff
@@ -109,19 +168,33 @@ struct Usage
             "do code coverage analysis"
         ),
         Option("cov=<nnn>",
-            "require at least nnn% code coverage"
+            "require at least nnn% code coverage",
+            `$(P Perform $(LINK2 $(ROOT_DIR)code_coverage.html, code coverage analysis) and generate
+            $(TT .lst) file with report.)
+---
+dmd -cov -unittest myprog.d
+---
+            )`,
         ),
         Option("D",
-            "generate documentation"
+            "generate documentation",
+            `$(P Generate $(LINK2 $(ROOT_DIR)spec/ddoc.html, documentation) from source.)
+            $(P Note: mind the $(LINK2 $(ROOT_DIR)spec/ddoc.html#security, security considerations).)
```

Comment:
I don't think that the paragraphs are needed here, but it's a 1:1 from dlang.org